.github/Issue_Template/Other.yml

```
@@ -0,0 +1,23 @@
+name: Other
+description: Use this for any other issues. Please do NOT create blank issues
+title: '[OTHER]'
+labels: ['🚦 status: awaiting triage','triage/accepted']
```

```suggestion
labels: ['triage/awaiting']
```
We don't have a `status: awaiting triage` label. Also we need to give `triage/awaiting` when issue is created. Maintainers will replace it with `triage/accepted` if the issue is accepted.